scalasql/core/src/Queryable.scala

```diff
@@ -68,6 +68,14 @@ object Queryable {
     var nulls = 0
     var nonNulls = 0
 
+    def restAreNulls(columnsCount: Int): Boolean = {
+      val result = Range.inclusive(index + 1, index + columnsCount).map { i =>
+        r.getObject(i) == null
+      }.forall(identity)
+      if(result) index = index + columnsCount
```

Comment:
This mutation is a bit surprising to me inside the method since the method name isn't suggestive of mutation.  I'm wondering if it should be moved to the call site instead.

Reply:
good idea